core/tests/gae_suite.py

```diff
@@ -28,10 +28,12 @@
 import os
 import sys
 import unittest
-#pylint: enable=wrong-import-order
 
 import feconf
 
+#pylint: enable=wrong-import-order
```

Comment:
Well... if it works without this then we don't need this at all, right?

Also, more generally, please make sure to verify all files manually after the changes have been made by isort.

Reply:
I will see this and manually check the files.